src/states/node.rs

```diff
@@ -62,15 +63,7 @@ const TICK_TIMEOUT_SECS: u64 = 60;
 /// Time (in seconds) after which a `GetNodeName` request is resent.
 const GET_NODE_NAME_TIMEOUT_SECS: u64 = 60;
 /// The number of required leading zero bits for the resource proof
-const RESOURCE_PROOF_DIFFICULTY: u32 = 10;
-
-// in Bytes
-fn resource_proof_target_size(min_size: usize, group_len: usize) -> u32 {
-    let evaluators = cmp::max(min_size, group_len);
-    // Default value: 2 MBytes sharing among the evaluators
-    // TODO: the algorithm need to be measured and refactored against the real network
-    2 * 1024 * 1024 / evaluators as u32
-}
+const RESOURCE_PROOF_DIFFICULTY: u8 = 4;
```

Comment:
i tried with 8 and it will fail many of our current tests (taking too much time to resolve the challenge)
also, I think this factor may needs to be refactored when test against the testnet, to figure out what's the most suitable value network can handle